R/pkg/R/group.R

```diff
@@ -243,17 +236,73 @@ setMethod("gapply",
           signature(x = "GroupedData"),
           function(x, func, schema) {
             try(if (is.null(schema)) stop("schema cannot be NULL"))
```

Comment:
with or without try both work work fine.

Without try the error looks like: 
 Error in .local(x, ...) : schema cannot be NULL 

with try: 
Error in try(if (is.null(schema)) stop("schema cannot be NULL")) : 
  schema cannot be NULL

Is there a convention in SparkR for showing an error message ?